### ISSUE TITLE
[suggestion] manual trigger class

### ISSUE THREAD

**Issue by zecar** [chinchang/hint.css#172]
I'm currently facing a issue with mobile devices

I want to have tooltips automatically disappear after 3 seconds. I'm toggling hint--always with js onClick and after 3 seconds to make it appear/disappear

The problem is that it remains visible because the element is :hovered. I can't have another class toggling the visibility/opacity of after&before because i'm sacrificing the animation

a possible solution would be implementing a .manual class with a toggle .is-hint-active class

any suggestion?

**Comment by chinchang** [comment]
You'll need to toggle the hint position classes also.
I tried creating a demo for you -> https://codepen.io/chinchang/pen/NmKbMB
Let me know if it works for you.

**Comment by zecar** [comment]
@chinchang yes, of course that works. But as I said, I don't want to sacrifice animation
